src/test/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryLocationLiveTest.java

```diff
@@ -236,12 +207,52 @@ public void testCreateInstanceOfNonSupportedPlan() {
         cloudFoundryPaasLocation.createServiceInstance(params.getAllConfig());
     }
 
+    @Test(groups = {"Live"})
+    public void testBindServiceToApplication() {
+        ConfigBag params = getDefaultApplicationConfiguration();
+        createApplicationStartAndCheck(params.getAllConfig());
+        createServiceAndCheck(getDefaultClearDbServiceConfigMap());
+        assertFalse(cloudFoundryPaasLocation.isServiceBoundTo(SERVICE_INSTANCE_NAME, applicationName));
+        cloudFoundryPaasLocation.bindServiceToApplication(SERVICE_INSTANCE_NAME, applicationName);
```

Comment:
Because, for `Cloud Foundry` an application and a service are different elements. And the client provide an method to bind an application and a service. You can bind both of them, using the following operation:
`````
getClient().services()
    .bind(BindServiceInstanceRequest.builder()
        .applicationName(applicationName)
        .serviceInstanceName(serviceInstanceName)
        .build())
`````
Where, just it is necessary that `applicationName` represents to an available application and `serviceInstanceName` an available service.